src/layman/rest_multipublication_test.py

```diff
@@ -181,6 +179,7 @@ def test_get_publications(publication_type, headers, query_params, expected_publ
     ({'order_by': 'gdasfda'}, (2, 400), {'parameter': 'order_by'}),
     ({'order_by': 'full_text'}, (48, 400), dict()),
     ({'order_by': 'bbox'}, (48, 400), dict()),
+    ({'order_by': 'title', 'ordering_bbox': '1,2,3,4'}, (48, 400), dict()),
```

Comment:
Add tests with point/line bounding box as ordering_bbox, bbox_filter, bounding box of publication.